.changeset/hungry-hornets-agree.md

```diff
@@ -0,0 +1,17 @@
+---
+"@comet/cms-admin": major
+"@comet/cms-api": major
+---
+
+Enhance CronJob Module
+
+-   CronJob page now shows lastest job run
+-   CronJob page allows to trigger cron jobs manually
+-   Subpage of CronJob page shows all job runs
+
+Warning: Only include this module if all your users should be able to trigger cron jobs manually or you have sufficient access control in place.
+
+Includes following changes to the public api:
+
+-   JobStatus is renamed to KubernetesJobStatus to avoid naming conflicts
+-   Rename BuildRuntime to JobRuntime
```

Comment:
```suggestion
Enhance CronJob module

-   Show latest job run on `CronJobsPage`
-   Add option to manually trigger cron jobs to `CronJobsPage`
-   Add subpage to `CronJobsPage` that shows all job runs

Warning: Only include this module if all your users should be able to trigger cron jobs manually or you have sufficient access control in place.

Includes the following breaking changes:

-   Rename `JobStatus` to `KubernetesJobStatus` to avoid naming conflicts
-   Rename `BuildRuntime` to `JobRuntime`
```

Reply:
f14c9a24a070b9b657536175d59228bb4ec51bc2